### DISCUSSION TITLE
Only ossec alerts are created

### DISCUSSION THREAD

**Discussion by andre7455** [Security-Onion-Solutions/securityonion#8377] [Unsupported Versions]
Hiii there everyone,

I'm new so this is probably a stupid question but I need to ask.
This is going to be a 2 parter just because I'm new and a noob.

I have a security Onion standalone running in a network and 1 virtual client added to it.
I've installed Wazuh installed and working (I know that because I'm getting login alerts from that device).
I have tried to install sysmon and beats, but I don't think it's working correct yet.

Enough preamble let's get in to the question: why do I only get alerts generated by ossec?
I have installed Zeek and Saricata with 1 instance each.

I would be so glad for just some pointing in the right direction

**Comment by reyesj2** [comment]
Are you actively ingesting data into Security Onion? You mentioned Zeek & Suricata are set to 1 process each. What is connected to your monitor/tap ports ?

**Reply by andre7455** [reply]
Just one single Windows 10 cliënt

**Reply by reyesj2** [reply]
In that case you will only see HIDS alerts (host based). Zeek and Suricata are NIDS (network) so you will need to tap a network connection via a physical TAP or span port on a switch.

If your standalone is connected to the internet you can run "sudo so-test" it will download another docker container and run pcap through your monitor ports so you can generate NIDS alerts.

**Reply by andre7455** [reply]
thank you so much! 
I was stuck and now I have another lead to investigate (probs need to add more
I will do that tomorrow first thing

**Reply by Grey999** [reply]
Did it work ? I got the same issue.

**Reply by andre7455** [reply]
Yes and no, let me explain.

I have imported a PCAP file I found in a tutorial and from there a few alerts were created from that.
but when I was going around to install a network tap to capture live network traffic my project was put on hold.

So I can't really say with 100% certainty that this is the solution but I'm about 90% certain about it.
